recipes/xerces-c/all/conanfile.py

```diff
@@ -99,7 +99,7 @@ def validate(self):
 
     def build_requirements(self):
         if self.options.message_loader == "icu" and not can_run(self):
-            self.tool_requires("icu/72.1")
+            self.tool_requires("icu/73.2")
```

Comment:
```suggestion
            self.tool_requires("icu/<host_version>")
```

But then the min required version should be something like `>=1.56.0 <2 || >=2.0.6`

Reply:
done